crates/store/src/db/schema.rs

```diff
@@ -0,0 +1,140 @@
+// @generated automatically by Diesel CLI.
+
+diesel::table! {
+    account_deltas (account_id, block_num) {
+        account_id -> Binary,
+        block_num -> BigInt,
+        nonce -> Integer,
+    }
+}
+
+diesel::table! {
+    account_fungible_asset_deltas (account_id, block_num, faucet_id) {
+        account_id -> Binary,
+        block_num -> BigInt,
+        faucet_id -> Binary,
+        delta -> Integer,
+    }
+}
+
+diesel::table! {
+    account_non_fungible_asset_updates (account_id, block_num, vault_key) {
+        account_id -> Binary,
+        block_num -> BigInt,
+        vault_key -> Binary,
+        is_remove -> Integer,
+    }
+}
+
+diesel::table! {
+    account_storage_map_updates (account_id, block_num, slot, key) {
+        account_id -> Binary,
+        block_num -> BigInt,
+        slot -> Integer,
+        key -> Binary,
+        value -> Binary,
+    }
+}
+
+diesel::table! {
+    account_storage_slot_updates (account_id, block_num, slot) {
+        account_id -> Binary,
+        block_num -> BigInt,
+        slot -> Integer,
+        value -> Binary,
+    }
+}
+
+diesel::table! {
+    accounts (account_id) {
+        account_id -> Binary,
+        network_account_id_prefix -> Nullable<BigInt>,
+        account_commitment -> Binary,
+        block_num -> BigInt,
+        details -> Nullable<Binary>,
+    }
+}
+
+diesel::table! {
+    block_headers (block_num) {
+        block_num -> BigInt,
+        block_header -> Binary,
+    }
+}
+
+diesel::table! {
+    note_scripts (script_root) {
+        script_root -> Binary,
+        script -> Binary,
+    }
+}
+
+diesel::table! {
+    notes (block_num, batch_index, note_index) {
+        block_num -> BigInt,
+        batch_index -> Integer,
+        note_index -> Integer,
+        note_id -> Binary,
+        note_type -> Integer,
+        sender -> Binary,
+        tag -> Integer,
+        execution_mode -> Integer,
+        aux -> Integer,
```

Comment:
`aux` can be up to 64 bits in size. Should we use `BitInt` here?

Reply:
Definitely!